stdlib/private/SwiftPrivateLibcExtras/Subprocess.c

```diff
@@ -0,0 +1,50 @@
+//===--- Subprocess.cpp - Subprocess Stubs --------------------------------===//
```

Comment:
No, I named it .c to avoid the extern "C" block.  I don't have a strong opinion on that, and can change it you believe it is better.